mix.exs

```diff
@@ -7,7 +7,8 @@ defmodule DisplayRecipe.MixProject do
       version: "0.1.0",
       elixir: "~> 1.9",
       start_permanent: Mix.env() == :prod,
-      deps: deps()
+      deps: deps(),
+      escript: [main_module: DisplayRecipe.CLI]
```

Comment:
Is the difference between this and using the `:mod` keyword in the `application` function below clear to you? When might you want to use that over this approach?